cli/connector/eval_plain_text.go

```diff
@@ -59,8 +60,10 @@ func evalPlainTextConn(conn net.Conn, funcBody string, args []interface{}, opts
 		return nil, err
 	}
 
+	buffer := bytes.Buffer{}
```

Comment:
Why do you declare buffer here? Seems that it's used only by `readFromPlainTextConn`.